airbyte-webapp/src/components/JobItem/components/Logs.tsx

```diff
@@ -57,4 +61,38 @@ const Logs: React.FC<LogsProps> = ({ logsArray }) => {
   );
 };
 
+/**
+ * Matching the log's line number by time makes the following assumptions:
+ * 1. The log's lines are already ordered by time
+ * 2. The timestamps used are in the same timezone
+ * 3. The error is closer to the end of the log file than the beginning
+ * 4. Lines are matched by the start of a line with "YYYY-MM-DD HH:mm:ss" format, like "2022-05-17 23:00:19 DEBUG I am a log message"
+ *
+ * TODO: it appears that LazyLog only highlights the first 2 lines in the array.  Can this be fixed?
```

Comment:
Could LazyLog be expecting a range instead of the individual line numbers? (e.g. `[10, 12]` instead of `[10, 11, 12]`)

Reply:
Genius!